### PR TITLE
 fix: use ">" instead of ">=" in resource allocation

### PR DESCRIPTION
**Pull request by draveness**
/kind cleanup
/sig scheduling

```release-note
NONE
```

### CODE REVIEW

File: pkg/scheduler/framework/plugins/noderesources/resource_allocation.go [hunk 1]
```diff
@@ -61,13 +61,13 @@ func (r *resourceAllocationScorer) score(
 	var score int64
 
 	// Check if the pod has volumes and this could be added to scorer function for balanced resource allocation.
-	if len(pod.Spec.Volumes) >= 0 && utilfeature.DefaultFeatureGate.Enabled(features.BalanceAttachedNodeVolumes) && nodeInfo.TransientInfo != nil {
```
Comment (Huang-Wei) [inline pkg/scheduler/framework/plugins/noderesources/resource_allocation.go:64]: I guess it's meant to be `len(pod.Spec.Volumes) > 0`?

File: pkg/scheduler/framework/plugins/noderesources/resource_allocation.go [hunk 2]
```diff
@@ -61,13 +61,13 @@ func (r *resourceAllocationScorer) score(
 	var score int64
 
 	// Check if the pod has volumes and this could be added to scorer function for balanced resource allocation.
```
Comment (damemi) [inline pkg/scheduler/framework/plugins/noderesources/resource_allocation.go:63]: if you're removing the volume check, should you update this comment too?

Comment (Huang-Wei) [reply pkg/scheduler/framework/plugins/noderesources/resource_allocation.go:63]: @damemi I suppose @draveness was trying to say the logic `>= 0` makes no sense as it's always true.

As mentioned in https://github.com/kubernetes/kubernetes/pull/95146#discussion_r496908079, checking `if len(volumes) > 0` might be the original intention.

Comment (draveness) [reply pkg/scheduler/framework/plugins/noderesources/resource_allocation.go:63]: > @damemi I suppose @draveness was trying to say the logic `>= 0` makes no sense as it's always true.

yes

> As mentioned in [#95146 (comment)](https://github.com/kubernetes/kubernetes/pull/95146#discussion_r496908079), checking `if len(volumes) > 0` might be the original intention.

Thanks, I'll update the commit here.

File: pkg/scheduler/framework/plugins/noderesources/resource_allocation.go (hunk 1 above)
Comment (draveness) [reply pkg/scheduler/framework/plugins/noderesources/resource_allocation.go:64]: done

**Pull request merged**